src/python/grpcio/grpc/_cython/_cygrpc/aio/iomgr/iomgr.pyx.pxi

```diff
@@ -12,7 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import asyncio
+try:
+    import asyncio
+except ImportError:
+    # TODO(https://github.com/grpc/grpc/issues/19728) Improve how Aio Cython is
+    # distributed without breaking none compatible Python versions. For now, if
+    # Asyncio package is not available we just skip it.
+    pass
+
```

Comment:
Nit: all these `.pxi` files will be concatenated into one, so we can import the libraries once in `cygrpc.pyx`. I didn't mention it before because it doesn't hurt performance and increases readability. But this case we don't want to duplicate TODO comment and create burden (one change apply to multiple places) for future changes.

Reply:
Do you want to import the `asyncio` module just in one place? where?

Reply:
Yes, that's what I mean. In the top of `cygrpc.pyx`, there is multiple imports.
https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/cygrpc.pyx#L19